scipy/stats/tests/test_morestats.py

```diff
@@ -677,6 +675,21 @@ def test_moments_normal_distribution(self):
         expected = [0.011315, 1.017931, 0.05811052, 0.0754134]
         assert_allclose(moments, expected, rtol=1e-4)
 
+        # test equivalence with `stats.moment`
+        m1 = stats.moment(data, moment=1)
+        m2 = stats.moment(data, moment=2)
+        m3 = stats.moment(data, moment=3)
+        assert_allclose((m1, m2, m3), expected[:-1], atol=1e-1)
```

Comment:
This is not the actual equivalence (docstring is wrong as I remarked on earlier). That's why you need ``atol=0.1`` to make the test pass:


    In [16]: stats.kstat(x, 3)
    Out[16]: -0.1788808700120989

    In [17]: stats.moment(x, 3)
    Out[17]: -0.12879422640871122